data/intro missions.txt

```diff
@@ -382,7 +382,7 @@ mission "Intro [2 Interceptor]"
 				`	"Got it."`
 					accept
 				`	"I really shouldn't try to fight?"`
-			`	"It just isn't feasible," he says. "If a ship has turrets, your Sparrow can't get close enough to hit it without being damaged, and then it's just a matter of whose weapons and shields are stronger. And one heavy rocket hit could nearly disable you. So, don't risk it."`
+			`	"It just isn't feasible," he says. "If a ship has turrets, your Sparrow can't get close enough to hit it with your guns without being damaged, and then it's just a matter of whose weapons and shields are stronger. And one heavy rocket hit could nearly disable you. So, don't risk it. And if it has anti-missile turrets, you can't count on hanging back and bombarding it with missiles from long distances, or at least not with weapons that can fit in a Sparrow. It's like rock paper scissors; recgonize what your enemy's strengths and weaknesses are, as well as your own."`
```

Comment:
Move `So, don't risk it.` before `It's like rock paper scissors` or just remove it.
`And if it has anti-missile turrets` -> `Also, if it has anti-missile turrets` (otherwise two sentences in close proximith both start with "And"